### PR TITLE
Cleanup ocp-staging cluster admins

### PR DESCRIPTION
**Pull request by knikolla**
Neither Leo nor Margaret are no longer on the project.

### CODE REVIEW

File: cluster-scope/overlays/ocp-staging/groups/local-cluster-admins.yaml [hunk 1]
```diff
@@ -6,4 +6,3 @@ metadata:
     kustomize.config.k8s.io/behavior: replace
 users:
   - mhaley@redhat.com
```
Comment (naved001) [inline cluster-scope/overlays/ocp-staging/groups/local-cluster-admins.yaml:8]: @larsks is `mhaley@redhat.com` part of the project?

**Review by naved001 — APPROVED** [review]
this looks good to me.

File: cluster-scope/overlays/ocp-staging/groups/local-cluster-admins.yaml (hunk 1 above)
Comment (larsks) [reply cluster-scope/overlays/ocp-staging/groups/local-cluster-admins.yaml:8]: No, that was Margaret (Red Hat co-op), who is no longer around.

Comment (knikolla) [reply cluster-scope/overlays/ocp-staging/groups/local-cluster-admins.yaml]: Updated the PR to remove the entire file.

**Review by naved001 — APPROVED** [review]

**Comment by naved001** [comment]
@larsks okay to merge this?

**Pull request merged**